layer/logged_data/image.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
```

Comment:
Please move this code under layer/contracts/logged_data.py

Reply:
Done